pom.xml

```diff
@@ -80,6 +80,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.opencsv</groupId>
+			<artifactId>opencsv</artifactId>
+			<version>4.2</version>
```

Comment:
What was the issue with opencsv 5.x? If there's a technical reason why we're using 4.x instead of 5.x, we should document it in case someone tries to update in the future.

Reply:
I forgot to change the version back to 5.1. Fixed.